src/syngen/ml/mlflow_tracker/mlflow_tracker.py

```diff
@@ -120,6 +121,8 @@ def start_run(
     ):
         if self.is_active:
             mlflow.start_run(run_id, experiment_id, run_name, nested, tags, description)
+            return mlflow.last_active_run()
```

Comment:
Please, create a separate method to fetch the last active run of mlflow. It might look like this code snippet - 
`def get_last_run():
    return mlflow.last_active_run()`

Reply:
Yes, it's one of the variant that hasn't come into production. 
But I think it's better to create get_last_run function.